### PR TITLE
DOC: Fix documentation and description for init_bold_grayords_wf

### LINKED ISSUE #3050: Documentation for init_bold_grayords_wf out of date
The documentation and description for `init_bold_grayords_wf` are out of date following @effigies refactoring in #3011. The documentation of the inputs currently refers to inputs that aren't used (`surf_files` and `surf_refs`) and doesn't refer to new inputs(`repetition_time` and `bold_fsLR`). Additionally, the description states that fsaverage was used as a high resolution intermediate when that is no longer the case.

### PR DESCRIPTION
**Pull request by Shotgunosine**
## Changes proposed in this pull request

Update documentation for `init_bold_grayords_wf`

## Documentation that should be reviewed
Removes mention of inputs that are no longer used (`surf_files` and `surf_refs`). Adds documentation of the new `repetition_time` parameter and `bold_fsLR` input. 

Additionally, updates the description to clarify that the surface data is transformed directly to fsLR and the surface data is transformed to MNI152NLin6Asym at the appropriate resolution.

Resolves #3050.

### CODE REVIEW

File: fmriprep/workflows/bold/resampling.py [hunk 1]
```diff
@@ -1268,16 +1269,18 @@ def init_bold_grayords_wf(
     from niworkflows.interfaces.utility import KeySelect
 
     workflow = Workflow(name=name)
+
+    mni_density = "2" if grayord_density == "91k" else "1"
```
Comment (mgxd) [inline fmriprep/workflows/bold/resampling.py:1273]: I believe the 170k grayordinates map out to ~ 1.6mm resolution
```suggestion
    mni_density = "2" if grayord_density == "91k" else "1.6"
```

**Review by mgxd — APPROVED** [review]
Thanks for catching that Dylan!

File: .zenodo.json [hunk 2]
```diff
@@ -330,6 +330,11 @@
       "name": "Blair, Ross W.",
       "orcid": "0000-0003-3007-1056"
     },
+    {
```
Comment (mgxd) [inline .zenodo.json:333]: This gets autogenerated, you'll have to add yourself to this file https://github.com/nipreps/fmriprep/blob/master/.maint/CONTRIBUTORS.md

Comment (Shotgunosine) [reply .zenodo.json:333]: Thanks, put the info in the right place this time.

File: fmriprep/workflows/bold/resampling.py (hunk 1 above)
Comment (mgxd) [reply fmriprep/workflows/bold/resampling.py:1273]: ah, i might just be misremembering. in any case, we can investigate after

**Pull request merged**